lint/style.py

```diff
@@ -38,27 +30,9 @@ def load_gutter_icons():
     if theme_files:
         theme_file = theme_files[0]
         opts = util.load_json(theme_file)
-        if not opts:
-            colorize = True
-        else:
-            colorize = opts.get("colorize", True)
-    else:
-        colorize = True
-
-    new_gutter_dict["colorize"] = colorize
-    dir_path, _ = os.path.split(theme_file)
-    pck_path = sublime.packages_path().split("/Packages")[0]
-    abs_dir = os.path.join(pck_path, dir_path)
-
-    png_files = glob(os.path.join(abs_dir, "*.png"))
-    for png in png_files:
-        png_file = os.path.basename(png)
-        name, ext = os.path.splitext(png_file)
-
-        new_gutter_dict["icons"][name] = os.path.join(dir_path, png_file)
-
-    global GUTTER_ICONS
-    GUTTER_ICONS = new_gutter_dict
+        if opts:
+            global COLORIZE
+            COLORIZE = opts.get("colorize", True)
```

Comment:
If you change from a 'colorize = False' Theme to `'none'`, it will stay ... 'False' 😄

Reply:
Oh duh, trying to be too efficient here